tests/Api/Message/MessageTest.php

```diff
@@ -150,4 +150,40 @@ public function testGetTimestamp()
 
         $this->assertEquals($datetime->getTimestamp(), $message->getTimestamp());
     }
+
+    public function testGetTtl()
+    {
+        $message = new Message("This is a test message");
+
+        $message->setTtl(3600);
+
+        $this->assertEquals(3600, $message->getTtl());
+    }
+
+    public function testSetTtl()
```

Comment:
The test is exactly duplicate to the one above. Maybe rename the text to canSetAndGetTtl() ?

Reply:
Good catch @OskarStark . Would you be able to open an issue please and I can fix it later? Or feel free to fix it if you like. Thanks for reviewing the PR!

Reply:
Always happy to help 🙌

Reply:
Thank you :)